crates/astria-composer/src/executor/bundle_factory/mod.rs

```diff
@@ -106,11 +106,21 @@ impl SizedBundle {
         mem::replace(self, Self::new(self.max_size))
     }
 
+    /// Returns the current size of the bundle.
+    pub(super) fn get_size(&self) -> usize {
+        self.curr_size
+    }
+
     /// Consume self and return the underlying buffer of actions.
     pub(super) fn into_actions(self) -> Vec<Action> {
         self.buffer
     }
 
+    /// Returns the number of sequence actions in the bundle.
+    pub(super) fn no_of_seq_actions(&self) -> usize {
```

Comment:
Hate the nomenclature, but let's go with it because the other parts of `bundle_factory` (we have `into_actions` and other methods, so let's not introduce `seq` because this might imply a difference between the `actions` in `into_actions` and `seq_actions`):
```suggestion
    pub(super) fn no_of_actions(&self) -> usize {
```

Also IMO this is cleaner to avoid abbreviations (write once, read many times):
```suggestion
    pub(super) fn actions_count(&self) -> usize {
```
or
```suggestion
    pub(super) fn number_of_actions(&self) -> usize {
```

Reply:
actions_count is good!